scripts/check-env.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
we need to check `go` is installed or not in the first place

Reply:
Will add that